Podcast/TagViewController.swift

```diff
@@ -134,8 +134,7 @@ class TagViewController: ViewController, UITableViewDelegate, UITableViewDataSou
     
     func recommendedSeriesTableViewCell(cell: RecommendedSeriesTableViewCell, didSelectItemAt indexPath: IndexPath) {
         
-        let seriesDetailViewController = SeriesDetailViewController()
-        seriesDetailViewController.series = series[indexPath.row]
+        let seriesDetailViewController = SeriesDetailViewController(series: series[indexPath.row])
```

Comment:
nice!

Reply:
There is probably a couple other places we do this, so please do a SEARCH to try to find them all